Project/SecondViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SecondViewController.swift
+//  Project
+//
+//  Created by Admin on 05.07.2021.
+//
+
+import UIKit
+
+class SecondViewController: UIViewController {
```

Comment:
```suggestion
final class SettingsViewController: UIViewController {
```
Имена не должны быть Second и тд